### ISSUE TITLE
[WCAG] Make footer bar WCAG

### ISSUE THREAD

**Issue by jolevesq** [Canadian-Geospatial-Platform/geoview#848]
The footer bar is not fully WCAG, make these adjustments

- [x] Increase text font size to 16px (do it from the theme so default text size is 16 for every component)
- [ ] Make sure tooltips are 16px font size
- [x] Add tooltip on: Coordinate button and scale button
- [x] Increase button size to 44px (see if we need to increase icon size)
- [x] Increase margin between element
- [x] Increase footer bar max size to 70px, increase footer bar min size to 40px
- [x] Aria-Label should be meaningful and use translation

![image](https://user-images.githubusercontent.com/3472990/214851399-6e1b817e-0e61-4a85-9fdb-f96e9967982c.png)

![image](https://user-images.githubusercontent.com/3472990/214852471-d5f61a52-d6f1-4b3a-91c3-a56c969a7427.png)
